tests/periph_i2c/tests/02__periph_i2c_write_register.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation       Data driven tests to verify writing to a valid register.
+
+Suite Setup         Run Keywords    Reset DUT and PHILIP
+...                                 DUT Must Have Periph I2C Application
+Test Setup          Reset DUT and PHILIP
```

Comment:
I think each test should include a DUT Must Have Periph I2C Application or at least send the I2C Get ID without checks.  Maybe make a clear buffers command that does that.

Reply:
so basically Test Setup should be the same as Suite Setup